packages/react/src/components/Checkbox/index.tsx

```diff
@@ -72,58 +72,62 @@ const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
 
     return (
       <>
-        <div className={classNames('Checkbox is--flex-row', className)}>
-          <input
-            id={id}
-            ref={typeof refProp === 'function' || !refProp ? checkRef : refProp}
-            type="checkbox"
-            checked={isChecked}
-            disabled={disabled}
-            onFocus={(e): void => {
-              setFocused(true);
-              if (typeof onFocus === 'function') {
-                onFocus(e);
+        <div className="column">
```

Comment:
I _think_ this looks like you're trying to wrap the checkbox to style it separately from the description, correct?

We have the odd ordering of elements here to support being able to target the label field following a checkbox. I'm wondering if we can use invert the order to allow for better text wrapping and instead use `:has` to target the appropriate elements. We would need to see how this would look in Firefox given that it doesn't yet fully support `:has` but we might be okay with some minor differences.